src/library_lz4.js

```diff
@@ -8,8 +8,8 @@
 addToLibrary({
   $LZ4__deps: ['$FS', '$preloadPlugins'],
   $LZ4: {
-    DIR_MODE: {{{ cDefs.S_IFDIR }}} | 511 /* 0777 */,
-    FILE_MODE: {{{ cDefs.S_IFREG }}} | 511 /* 0777 */,
+    DIR_MODE: {{{ cDefs.S_IFDIR }}} | 0o777,
```

Comment:
Does closure optimize this? The octal notation is easier to read but it is larger.

Reply:
The acorn optimizer might also optimize this, maybe just by default, if it doesn't track what was octal originally and preserve it?

Reply:
Yes, which ever layer it is that converts this to a normal number, it is happening since this change has zero code size changes.